resolver/GlobalPass.cc

```diff
@@ -191,14 +192,18 @@ void Resolver::finalizeAncestors(core::GlobalState &gs) {
     for (int i = 1; i < gs.classAndModulesUsed(); ++i) {
         auto ref = core::SymbolRef(&gs, core::SymbolRef::Kind::ClassOrModule, i);
         ENFORCE(ref.data(gs)->isClassOrModule());
-        auto loc = ref.data(gs)->loc();
-        if (loc.file().exists() && loc.file().data(gs).sourceType == core::File::Type::Normal) {
-            classCount++;
-        }
         if (!ref.data(gs)->isClassModuleSet()) {
             // we did not see a declaration for this type not did we see it used. Default to module.
             ref.data(gs)->setIsModule(true);
         }
+        auto loc = ref.data(gs)->loc();
+        if (loc.file().exists() && loc.file().data(gs).sourceType == core::File::Type::Normal) {
+            if (ref.data(gs)->isClassOrModuleClass()) {
```

Comment:
Need to be done after line 197 of we will enforce on a symbol not a class nor a module.